tensorflow/python/BUILD

```diff
@@ -5739,6 +5739,9 @@ cuda_py_test(
         "//tensorflow/python/data/ops:dataset_ops",
     ],
     xla_enable_strict_auto_jit = True,
+    # Sporadic permissions issues on Windows related to moving/deleting temp dirs.
+    # This test is unrelated to DML usage.
+    tags = ["no_dml"],
```

Comment:
Could we use the no_windows tag instead? If it's unrelated to Windows and not specific to DML, we'd want to still run them in Linux (and presumably they would also fail in CPU and GPU builds on Windows).

Reply:
Same errors even when running upstream version of TF. It has nothing to do with DML. I didn't do "no_windows" to make it easier to look up tests we intentionally disabled in this fork, but since this won't merge I suppose it doesn't matter.